app/inputs/with_accessible_help.rb

```diff
@@ -6,10 +6,10 @@ def label(wrapper_options = nil)
     text
   end
 
-  def hint(wrapper_options = nil)
-    text = translate_from_namespace(:metadata_help)
-    return unless text
-    template.content_tag 'p', text, class: 'field-help'
-  end
+  # def hint(wrapper_options = nil)
+  #   text = translate_from_namespace(:metadata_help)
+  #   return unless text
+  #   template.content_tag 'p', text, class: 'field-help'
+  # end
```

Comment:
Leaving breadcrumb pending final UX review, if UX needs something more complicated than what can be configured in the simple_form.rb initializer.